docs/core/policies.rst

```diff
@@ -417,6 +417,26 @@ training data. It predicts the next action with confidence ``1.0``
 if this exact conversation exists in the training data, otherwise it
 predicts ``None`` with confidence ``0.0``.
 
+Augmented Memoization Policy
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+The ``AugmentedMemoizationPolicy`` remembers examples from training
+stories for up to ``max_history`` turns. It has a forgetting mechanism
+that will forget a certain amount of steps in the conversation history
+and try to find a match in your stories with the reduced history. It
+predicts the next action with confidence ``1.0`` if a match is found,
+otherwise it predicts ``None`` with confidence ``0.0``.
+
+.. note::
+  
+  If it is needed to recall turns from training dialogues where some
+  slots might not be set during prediction time, add relevant stories
+  without such slots to training data, e.g. reminder stories. Since 
+  slots that are set some time in the past are preserved in all
+  future feature vectors until they are set to None, this policy has
+  a capability to recall the turns up to ``max_history`` from training
+  stories during prediction even if additional slots were filled in
+  the past for current dialogue.
```

Comment:
Could you please make it a little bit shorter

Reply:
I'm gonna give some more specific feedback on that :D

Reply:
I would remove this part, since it essentially just repeats what is said above: 
```
Since 
  slots that are set some time in the past are preserved in all
  future feature vectors until they are set to None, this policy has
  a capability to recall the turns up to ``max_history`` from training
  stories during prediction even if additional slots were filled in
  the past for current dialogue."
```

Reply:
And I would rephrase the other part of the note to this:
```
If you have dialogues where some slots may not be set during prediction time (e.g. when using reminders), make sure to add the relevant stories without slots to your training data as well. 
```

Reply:
@Ghostvv let me know what you think

Reply:
I think it's wise versa: during prediction time, there are more slots than during training. This policy can forget these additional slots and recall training story

Reply:
```
If it is needed to recall turns from training dialogues where some
  slots might not be set during prediction time, add relevant stories
  without such slots to training data, e.g. reminder stories
```
that seems to indicate something different? this is copied directly out of our source code

Reply:
yes, so `some slots might not be set` in training stories compared to `prediction time` 😅

Reply:
We probably want to rephrase there as well

Reply:
can you rephrase the parts of my description you wanted to rephrase and post here? i'm confused about which wording we're changing now 🙈

Reply:
```
If you have dialogues where some slots that are set during prediction time might not be set in training stories (e.g. in training stories starting with a reminder not all previous slots are set), make sure to add the relevant stories without slots to your training data as well.
```